packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -54,6 +58,31 @@ abstract class FlutterTestRunner {
     String? integrationTestUserIdentifier,
     TestTimeRecorder? testTimeRecorder,
   });
+
+  /// Runs tests using the experimental strategy of spawning each test in a
+  /// separate lightweight Engine.
+  Future<int> runTestsBySpawningLightweightEngines(
```

Comment:
I've omitted some parameters of `runTests` that didn't seem to make sense to support here. Are there any others that are worth omitting? Maybe `concurrency`?